docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -151,6 +151,20 @@ The deprecated `universal_discovery_widget_module.default_location_id` setting h
 
 - Deprecated `SubtreeQuery` class has been removed. In v3.0 it was replaced by `\EzSystems\EzPlatformAdminUi\QueryType\SubtreeQueryType`.
 
+#### View matching
+
+When matching views using custom services, the services must be now tagged with `ezplatform.view.matcher`.
+The matching must be configured in the following way:
+
+``` yaml
+content_view:
+    full:
+        folder:
+            template: folder.html.twig
+            match:
+                '@App\Matcher\MyMatcher': 2
```

Comment:
Maybe we can also change it to `'@App\Matcher\MyMatcher': ~` because `2` is not really important here and might be confusing.